2024/survey.yaml

```diff
@@ -356,6 +356,19 @@ questions:
       - Wiki (unofficial)
       - Blog posts
       - Others
+  - prompt: If you want to contribute or have contributed in the past to Nixpkgs, please select all that apply:
+    type: multiple
+    choices:
+      - I don't plan to contribute.
+      - Everything I want to contribute is already being worked on and I don't feel the need to get involved
+      - I don't know how to get started.
+      - My experience contributing was as expected.
+      - I tried to contribute something but found it hard to get started.
```

Comment:
Is there a difference between "I don't know how to get started" and "I tried to contribute something but found it hard to get started"?

Reply:
Almost the same, but one shows that you tried and got stuck, the other that even trying had a too high barrier.